### ISSUE TITLE
Low priority - standard messages pdf in user interface

### ISSUE THREAD

**Issue by jjj-unikent** [unikent/programmes-plant#326]
Hi I just noticed that the standard messages pdf text that shows under overridable fields doesn't link to anything.  If we supply you with the pdf can you add it in as and when?  Thanks JJ
ps Angela's on leave for a while so there might be a wait on some of your other questions.
